src/main/java/org/ehrbase/fhirbridge/ehr/converter/PatientCompositionConverter.java

```diff
@@ -0,0 +1,128 @@
+        composition.setStartTimeValue(composition.getAlter().getTimeValue());
+
+        // Required fields by API
+        composition.setLanguage(Language.DE);
+        composition.setLocation("test"); //FIXME: sensible value
+        composition.setSettingDefiningCode(Setting.SECONDARY_MEDICAL_CARE);
+        composition.setTerritory(Territory.DE);
+        composition.setCategoryDefiningCode(Category.EVENT);
+        composition.setComposer(new PartySelf()); //FIXME: sensible value
+
+        return composition;
+    }
+
+    private List<EthnischerHintergrundCluster> get_ethnic_background_data(Patient fhir_patient) {
+        List<EthnischerHintergrundCluster> items = new ArrayList<>();
+        try {
+            Extension extension_ethnic_group = fhir_patient.getExtensionByUrl("https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/ethnic-group");
+            Coding ethnic_group = (Coding) extension_ethnic_group.getValue();
+            EthnischerHintergrundCluster ec = new EthnischerHintergrundCluster();
+            ec.setEthnischerHintergrundDefiningCode(EthnischerHintergrundDefiningCode.get_by_SNOMED_code(ethnic_group.getCode()));
+            items.add(ec);
+        } catch (Exception e) {
+            throw new UnprocessableEntityException(e.getMessage());
+        }
+        return items;
+    }
+
+    private AlterObservation get_age_from_fhir(Patient fhir_patient) {
+        AlterObservation age = new AlterObservation();
+        try {
+            Extension extension_age = fhir_patient.getExtensionByUrl("https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/age");
+            //age - date_time_of_documentation
+            DateTimeType date_time_of_documentation_dt = (DateTimeType) extension_age.getExtensionByUrl("dateTimeOfDocumentation").getValue();
+            ZonedDateTime date_time_of_documentation = date_time_of_documentation_dt.getValueAsCalendar().toZonedDateTime();
+            //age - value
+            Age age_value = (Age) extension_age.getExtensionByUrl("age").getValue();
+            //age - Ereigniszeitpunkt
+            age.setOriginValue(date_time_of_documentation);
+            age.setTimeValue(date_time_of_documentation);
+            //age - Alter (ISO8601 duration e.g. P67Y)
+            age.setAlterValue(Period.ofYears(age_value.getValue().intValue()));
+            age.setSubject(new PartySelf());
+            age.setLanguage(Language.DE);
+        } catch (Exception e) {
+            throw new UnprocessableEntityException(e.getMessage());
+        }
+        return age;
+    }
+
+    private DatenZurGeburtCluster get_data_on_birth(Patient fhir_patient) {
+        DatenZurGeburtCluster datenZurGeburtCluster = new DatenZurGeburtCluster();
+        try {
+            //date of birth
+            datenZurGeburtCluster.setGeburtsdatumValue(fhir_patient.getBirthDate().toInstant().atZone(ZoneId.of("Europe/Berlin")).toLocalDate());
+        } catch (Exception e) {
```

Comment:
Do you really watn to catch ANY Exception here

Reply:
...yes, I guess you don't like that? I have no clue how a more elegant solution could look like. I added a hint on what happened since tests revealed I did not get any sensible message before.

Reply:
see what exception you are catching here and change Exception to exact exception.